src/components/IconWithTooltip/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { OverlayTrigger, Tooltip } from '@edx/paragon';
+import useWindowSize from '../../hooks/useWindowSize';
+
+
+const IconWithTooltip = ({
+  icon, altText, tooltipText, placementSm = 'right', placementLg = 'top', trigger = ['hover', 'focus'], breakpoint = 768, iconClassNames = 'ml-1',
```

Comment:
not sure if eslint cares, but perhaps better to have one prop per line?

Reply:
Eslint does not care, but yeah, one per line can be nice. 👍

Reply:
for me prettier usually takes care of this